src/context/errors.ts

```diff
@@ -5,8 +5,6 @@ import { json } from './json'
 /**
  * Returns a list of GraphQL errors.
  */
-export const errors = (
-  errorsList: Partial<GraphQLError>[],
-): ResponseTransformer<{ errors: typeof errorsList }> => {
-  return json({ errors: errorsList })
+export const errors = (errorsList: Partial<GraphQLError>[]) => {
```

Comment:
We need to be careful in dropping the `ResponseTransformer` return type here. It may cause some unexpected type issues when using `ctx.errors` in combination with other utils.

Reply:
Ok. Thanks.